action.yml

```diff
@@ -12,15 +12,17 @@ inputs:
     description: "Config file location. If not present, the default 'release-plz.toml' is used."
     required: false
   manifest_path:
-    description: "Path to the Cargo.toml of the project you want to update. If not provided, release-plz will use the Cargo.toml of the
-            root directory. Both Cargo workspaces and single packages are supported."
+    description:
+      "Path to the Cargo.toml of the project you want to update. If not provided, release-plz will use the Cargo.toml of the
+      root directory. Both Cargo workspaces and single packages are supported."
     required: false
   project_manifest:
     description: "Deprecated. Use `manifest_path` instead."
     required: false
   version:
+    # TODO fix this
     description: "Release-plz version to use. It must be an existing git tag name. For example `release-plz-v0.2.45`. (Default: `latest`)."
-    default: "release-plz-v0.3.70"
+    default: "0.3.70"
```

Comment:
So this is already a bit weird. You're making a release tool, for releasing crate versions, then publishing that tool to crates.io, but then installing it from your git repo via a specific tag version...

I suppose this makes sense if you're wanting to test out new versions of the tool in an action against a not yet released version of release-plz. I'm not sure if binstall supports some sort of git-tag version specifier.

I suspect this might break some small amount of workflows that specify this, but maybe the logic below can detect this and chop out the release-plz-v prefix somehow?

Reply:
> but then installing it from your git repo via a specific tag version

yes, this is because I was using an action that wasn't integrated with crates.io.

> I suspect this might break some small amount of workflows

I'm not aware of any. This new approach should work well :)

For testing, typically I use `cargo install` because I don't tag "dev" versions in release-plz repo

Reply:
> I suspect this might break some small amount of workflows that specify this,

Now that I think about it. Yes, I think I broke some people GitHub actions. This should have been a 0.6.0 D: